src/tools/parse_simulation.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
This line causes the script to file for users who don't have their interpreter in /usr/bin/env